ml-testing-toolkit/Chart.yaml

```diff
@@ -36,11 +36,6 @@ dependencies:
     version: 11.1.0
     repository: "file://./chart-connection-manager-backend"
     condition: connection-manager-backend.enabled
-  - name: percona-xtradb-cluster
```

Comment:
We may need another schema added to the MySQL server specfically for the TTK here.

Reply:
@mdebarros : ahh , ok I will looking into that.  ..let me get back to you.

Reply:
I restored back as discussed but changed percona -> mysql as that percona chart is still problematic